src/deadline/client/cli/_common.py

```diff
@@ -65,13 +66,11 @@ def wraps(ctx: click.Context, *args, **kwargs):
         except click.ClickException:
             # Let click exceptions fall through
             raise
-        except Exception as e:
+        except Exception:
             # Log and print out unfamiliar exceptions with additional
             # messaging.
-            click.echo(f"The AWS Deadline Cloud CLI encountered the following exception:\n{e}")
-
-            if logging.DEBUG >= logger.getEffectiveLevel():
-                logger.exception("Exception details:")
+            click.echo("The AWS Deadline Cloud CLI encountered the following exception:")
+            click.echo(traceback.format_exc())
```

Comment:
Thank you for this one, I was trying to debug something last night and the limited messaging was driving me nuts.